components/CatalogSearch.vue

```diff
@@ -152,4 +175,21 @@
             display: none;
         }
     }
+
+    @include media-breakpoint-up(lg) {
+        .filter {
+            display: flex;
+            gap: $spacer-2;
+            flex-wrap: wrap;
+            &__pill {
+                margin-top: $spacer-2;
+                cursor: pointer;
+                font-size: map-get($font-size-phone, 5);
```

Comment:
Please use the class p5 instead of defining it here. Add the class p5 to this element, by doing so the font-size is scaling responsive.